mapflatgeobuf.c

```diff
@@ -167,11 +167,13 @@ int msFlatGeobufLayerOpen(layerObj *layer)
   {
     OGRSpatialReferenceH hSRS = OSRNewSpatialReference( NULL );
     char *pszWKT = NULL;
-    if (ctx->srid) {
+    if (ctx->srid > 0) {
       if (!(OSRImportFromEPSG(hSRS, ctx->srid) != OGRERR_NONE))
         return MS_FAILURE;
       if( OSRExportToWkt( hSRS, &pszWKT ) != OGRERR_NONE)
         return MS_FAILURE;
+    } else if (ctx->wkt == NULL) {
+      return MS_SUCCESS;
```

Comment:
OSRDestroySpatialReference(hSRS); should be called here, and in the 2 above return MS_FAILURE cases